src/Api/IriConverterInterface.php

```diff
@@ -44,7 +44,7 @@ public function getItemFromIri(string $iri, bool $fetchData = false);
      *
      * @return string
      */
-    public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH) : string;
+    public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::CONFIG) : string;
```

Comment:
Why cannot we let the current default value? The default value will be injected everywhere by the DIC anyway, but it will let users using components directly (without Symfony) to have sensible defaults.